common/common-rest/src/main/java/io/servicecomb/common/rest/codec/param/FormProcessorCreator.java

```diff
@@ -69,6 +75,54 @@ public FormProcessorCreator() {
   @Override
   public ParamValueProcessor create(Parameter parameter, Type genericParamType) {
     JavaType targetType = TypeFactory.defaultInstance().constructType(genericParamType);
+
+    if (isPart(parameter)) {
+      return new PartProcessor(parameter.getName(), TypeFactory.defaultInstance().constructType(String[].class));
+    }
     return new FormProcessor(parameter.getName(), targetType);
   }
+
+  private boolean isPart(Parameter parameter) {
+    return new FileProperty().getType().equals(((FormParameter) parameter).getType());
+  }
+
+  private static class PartProcessor extends AbstractParamProcessor {
```

Comment:
below code is enough:

    @Override
    public Object getValue(HttpServletRequest request) throws Exception {
      return request.getPart(paramPath);
    }

    @Override
    public void setValue(RestClientRequest clientRequest, Object arg) throws Exception {
      clientRequest.attach(paramPath, (Part) arg);
    }